lineage/CellVar.py

```diff
@@ -95,6 +110,11 @@ def isRootParent(self):
     def get_sister(self):
         """
         Member function that gets the sister of the current cell.
+
+        :param self: Instance of the cell object
+        :type self: object
```

Comment:
same. In these cases I guess you can just define the output, or what an instance does.